File: Ygo_Deck_Helper/Card.cs
namespace Ygo_Deck_Helper
{
    internal class Card
    {
        private int Card_Id;
        private int Type_Number;
        private string Card_Name;

        /// <summary>
        /// Creates new card object
        /// </summary>
        /// <param name="Passcode">Konami ID of card </param>
        /// <param name="Type_Number">Ygo Pro has specific numbers assigned based of the type of card </param>
        /// <param name="Card_Name">Official Konami Name of card  </param>
        public Card(int Passcode, int Type_Number, string Card_Name)
        {
            this.Card_Id = Passcode;
            this.Type_Number = Type_Number;
            this.Card_Name = Card_Name;
        }

        public int Get_Card_Id()
        {
            return Card_Id;
        }

        public int Get_Card_Type_Number()
        {
            return Type_Number;
        }

        public string Get_Card_Name()
        {
            return Card_Name;
        }
    }
}

File: Js_Version/Card.js
/*jshint es6: True */
'use strict';

var Card = function() {

	function Card(Card_Id, Type_Number, Card_Name) {
        this.mCard_Id  = Card_Id;
        this.mType_Number = Type_Number;
        this.mCard_Name = Card_Name;
	}
	
Card.prototype.Get_card_id = function() {
	
	return this.Get_card_id;
}

Card.prototype.Get_TypeNumber  = function() {
	return this.Get_TypeNumber;
}

Card.prototype.Get_card_name  = function() {
	return this.mCard_Name;
}
	
}();
